stylesheet.css

```diff
@@ -218,6 +219,7 @@ table tr {
   border: 1px solid #ccc;
 }
 form {
+  margin-top: 0px;
   margin-top: 10px;
```

Comment:
You have added to margin-top properties.

Reply:
fixed!